pelican/writers.py

```diff
@@ -140,11 +140,12 @@ def write_feed(self, elements, context, path=None, feed_type='atom',
                 complete_path, context=context, feed=feed)
         return feed
 
-    def write_file(self, name, template, context, relative_urls=False,
+    def write_file(self, name, url, template, context, relative_urls=False,
```

Comment:
This is quite a change and could possibly break some plugins. Did you do any checks? Somewhat *nicer* way would be making this parameter kwarg.

Reply:
Made `url` into an optional parameter, defaulting to `None`. Is that okay? :)

Reply:
Yeah, should be fine now. Thanks.